### PR TITLE
feat(20100): e2e org switching

### PR DESCRIPTION
**Pull request by nnorbert**
E2E organisation switching

### CODE REVIEW

**Review by rgrnearform — DISMISSED** [review]
lgtm

**Review by peschina — DISMISSED** [review]
LGTM, a few minor comments

File: apps/payments/e2e/fixtures/auth.ts [hunk 1]
```diff
@@ -1,97 +1,139 @@
...
-      const id = test.info().parallelIndex;
-      const fileName = path.resolve(
-        test.info().project.outputDir,
-        `.auth/public-servant-${id}.json`,
-      );
+const getWorkerStorageState = async (
+  browser: Browser,
+  storagePath: string,
+  userName: string,
+  pass: string,
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/payments/e2e/fixtures/auth.ts]: Done

File: apps/payments/e2e/fixtures/pageFixtures.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { Page } from "@playwright/test";
+import { test as base } from "./auth";
+
+type citizenPageFixtures = {
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/payments/e2e/fixtures/pageFixtures.ts]: Done

File: apps/payments/e2e/objects/components/OrganisationSelector.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+  async isNotVisible() {
+    await expect(this.form).not.toBeVisible();
+  }
+
+  async isActive() {
+    const selectDisabled = await this.orgSelector.isDisabled();
+    const buttonDisabled = await this.submitBttn.isDisabled();
+    await expect(selectDisabled).toBeFalsy();
+    await expect(buttonDisabled).toBeFalsy();
```
[the remark this answers is not in the document]
Comment (nnorbert) [reply apps/payments/e2e/objects/components/OrganisationSelector.ts]: Done

**Review by peschina — APPROVED** [review]

**Pull request merged**